src/app/component/liquidation/LiquidationModal.tsx

```diff
@@ -0,0 +1,103 @@
+  const [collateralAmount, setCollateralAmount] = useState("");
+
+  const handleCollateralOptionChange = (
+    e: React.ChangeEvent<HTMLSelectElement>
+  ) => {
+    const selectedValue = e.target.value;
+    const selectedOption = collateralAmountOptions.find(
+      (option) => option.value === selectedValue
+    );
+    setSelectedCollateralAmountOption(selectedOption || null);
+  };
+
+  return (
+    <Modal showModal={open} toggleModal={handleClose}>
+      <div className="w-[330px] pt-5 px-6 flex flex-col gap-6">
+        <span className="text-neutral-800 text-lg font-semibold font-['Raleway'] leading-normal">
+          Liquidate position
+        </span>
+        <div className="request-amount flex flex-col gap-2 relative w-full mt-[10px]">
+          <h4 className="text-textBlack text-base font-medium font-['Inter'] leading-tight tracking-tight">
+            Replay Amount
+          </h4>
+          <div className="bg-gray-100 px-2 py-4 flex rounded-2xl relative">
+            <input
+              type="text"
+              placeholder="1,000.04"
+              className="text-textBlack bg-transparent border border-transparent w-[55%] focus:outline-none"
+              value={collateralAmount}
+              onChange={(e) => setCollateralAmount(e.target.value)}
+            />
+            <div className="w-[45%] flex gap-4 justify-end self-center relative">
+              <span className="h-full text-[#C3C8CA]">{"|"}</span>
+              <div className="flex justify-center gap-2 items-center relative">
+                <AiOutlineDollarCircle
+                  size={24}
+                  className="inline text-appColor1"
+                />
+                <select
+                  name="requestAmount"
+                  id="requestAmount"
+                  // className="bg-transparent cursor-pointer focus:border-transparent outline-none"
+                  className="block w-full text-sm bg-transparent appearance-none  focus:outline-none focus:ring-0 peer"
+                  onChange={handleCollateralOptionChange}
+                >
+                  {collateralAmountOptions.map((opt, idx) => (
+                    <option key={idx} value={opt.value}>
+                      {opt.name}
+                    </option>
+                  ))}
+                </select>
+              </div>
+            </div>
+          </div>
+        </div>
+        <hr />
+        <div className="py-3">
+          <LoanSummaryContainer title="Summary">
+            <LoanSummary title="Health Factor" amount={`40  ${"%"}`} />
+            <LoanSummary title="Borrowing amount" amount={`3000,04 USDC`} />
+            <LoanSummary title="Your collateral" amount={`2,500,50 USDC`} />
+            <LoanSummary title="Colateral rate" amount={`${150}%`} />
+            <LoanSummary title="Platform Fee" amount={`3.34 USDC`} />
+            <LoanSummary title="Amount" amount={`386.95 USDC`} />
+          </LoanSummaryContainer>
```

Comment:
Why we don't fetch real data here? even if is dummy?

Reply:
Provide me api i will do real data. This milestone is only for UI fixes not for real data implementation or api etc